### PR TITLE
Add Uyuni GPG to the database

### PR DESCRIPTION
**Pull request by juliogonzalez**
## What does this PR change?

Add Uyuni GPG to the database

## GUI diff

No difference.

- [x] **DONE**

## Documentation
- No documentation needed: Bugfix

- [x] **DONE**

## Test coverage
- No tests: Bugfix

- [x] **DONE**

## Links

Tracks # https://github.com/SUSE/spacewalk/issues/6374

## Changelogs

Copy the following sentence as a new comment if you don't need changelog entries:

> gitarro no changelog needed !!!

If the test `changelog_test`already ran, then add another new comment with the following text:

> gitarro rerun changelog_test !!!

### CODE REVIEW

**Comment by juliogonzalez** [comment]
Do I need a migration script as well?

**Comment by mcalmer** [comment]
Yes, migration script is needed as well

**Comment by juliogonzalez** [comment]
Done, but I wonder: what's the point of having this at `schema/spacewalk/common/`

IIUC:

- For users with Uyuni already installed, the scripts at `schema/spacewalk/common/` will not run, and they will get the GPG key ingested thanks to the migration script.
- For users that are installing from scratch, `schema/spacewalk/common/` will run, but anyway the migration script will run as well.

Let's discuss it tomorrow, maybe there's something I am missing about how the DB migration tool works.

**Comment by juliogonzalez** [comment]
I added the provider as well.

File: schema/spacewalk/upgrade/susemanager-schema-4.0.2-to-susemanager-schema-4.0.3/300-add-uyuni-keys.sql [hunk 1]
```diff
@@ -0,0 +1,2 @@
+insert into rhnPackageProvider (id, name) (select sequence_nextval('rhn_package_provider_id_seq'), 'Uyuni' from dual where not exists (select 1 from rhnPackageProvider where name = 'Uyuni'));
```
Comment (mcalmer) [inline schema/spacewalk/upgrade/susemanager-schema-4.0.2-to-susemanager-schema-4.0.3/300-add-uyuni-keys.sql]: Good, but this is now also needed in common/data/

Comment (juliogonzalez) [reply schema/spacewalk/upgrade/susemanager-schema-4.0.2-to-susemanager-schema-4.0.3/300-add-uyuni-keys.sql]: Done. I had it ready yesterday, but did not remember to add it to the commit.

**Pull request merged**